apps/chatbot/src/modules/vector_database.py

```diff
@@ -52,6 +52,7 @@
     aredis.ConnectionPool.from_url(REDIS_URL)
 )
 REDIS_INDEX_NAME = os.getenv("CHB_REDIS_INDEX_NAME")
+INDEX_ID = os.getenv("CHB_LLAMAINDEX_INDEX_ID")
```

Comment:
To avoid modifying the entire set of environment variables during automatic indexing, I would use an ssm parameter.
This way, the CHB_LLAMAINDEX_INDEX_ID would contain the name of the ssm parameter. 

NB: it needs the import of the get_ssm_parameter method
```suggestion
INDEX_ID = get_ssm_parameter(os.getenv("CHB_LLAMAINDEX_INDEX_ID"))
```

Reply:
ok